tests/test_radosbench.py

```diff
@@ -0,0 +1,29 @@
+import os
+from kubernetes import client, config
+os.sys.path.append(os.path.dirname(os.getcwd()))
+
+from oc import pod, pod_exec
+from tasks import radosbench
+
+
+def run():
+    config.load_kube_config()
+    v1 = client.CoreV1Api()
+    ret = v1.list_pod_for_all_namespaces(watch=False,
+                                        label_selector='app=rook-ceph-tools')
+    for i in ret.items:
+        namespace = i.metadata.namespace
+        name = i.metadata.name
+        break
+
+    po = pod.Pod(name, namespace, roles=['client'])
+    radosbench.run(ceph_pods=[po],
+                   config={
+                       'time': 10,
+                   })
+
+
+if __name__ == "__main__":
```

Comment:
Lets remove these 2 lines, since the import will happen in runner.

Reply:
Done, made test_bench as test script for test_radosbench